README.md

```diff
@@ -38,36 +49,56 @@ docker-compose --version
 - Fast, consistent delivery of your applications
 - Responsive deployment and scaling
 - Running more workloads on the same hardware
+- Not having to worry about package dependencies
 
 <a id="5"></a>
-### 5. Setup 
-Run the following commands in your terminal to build and deploy the JupyterLab container:
+### 5. Build Jupyter Lab Container 
+First, clone this repository on your remote server/virtual machine (vm) or local by using `https` or `ssh` or `Github CLI`. 
+
+Then, run the following commands in your terminal to build and deploy the JupyterLab container:
 ```
 cd docker101_jupyterlab
-docker-compose up --build
+docker compose up --build
 ```
+The above step extracts and downloads the specified `JupyterLab` image onto your VM or local and then spins it onto a running instance called as the container. 
+
 Use `CTRL + C` to exit out of the container at any time. 
 
-To run the container in detached mode add `-d` as follows
+To run the container in detached mode so that you can reuse the existing terminal, please add `-d` while building the image as follows
+```
+docker compose up --build -d
+```
+You could also run 
+```
+docker compose up -d
+```
+This will just reuse the image if it already exists on your VM or local, If not, it will extract again. But `--build` forces the docker to rebuild the image every time it's run. This is helpful and must when you are adding new requirements to your container.  
+
+If you have multiple containers listed on the `Docker Compose` file and you wish to start only the `JupyterLab` container, run the above command with the desired `Docker` service name as follows. 
 ```
-docker-compose up --build -d
+docker compose up -d jupyter101
 ```
 
-If you have successfully built and deployed the JupyterLab image container using either of the above commands, you can access the web interface of the JupyterLab at 
+If you have successfully built and deployed the JupyterLab image container using the above commands, you can access the web interface of the JupyterLab at 
 ```
-http://localhost:8888
+http://localhost:8777
```

Comment:
I'm maybe doing this part wrong but if you're not using a terminal in a remote ssh vscode session or something, you'll need to manually forward the port. In my case it forwarded to port 8888.